### PR TITLE
Nwd nnw

### PR DESCRIPTION
**Pull request by pwiatr3**

### CODE REVIEW

**Comment by ziobron** [comment]
✅ CI Passed - Task: nwd-nnw
🏅 Granted 6 XP for @pwiatr3

**Comment by ziobron** [comment]
⏱ 3 XP extra for @pwiatr3 for on-time delivery

File: homework/nwd-nnw/nwdNww.hpp [hunk 1]
```diff
@@ -1,11 +1,35 @@
...
-}
-
-int NWW(int lhs, int rhs) {
-    // TODO: Implement me :)
-    return -1;
-}
+#pragma once
+
+int NWD(int lhs, int rhs) {
+    // TODO: Implement me :)
```
Comment (nauka-programowania-MA) [inline homework/nwd-nnw/nwdNww.hpp:4]: remove

File: homework/nwd-nnw/nwdNww.hpp [hunk 2]
```diff
@@ -1,11 +1,35 @@
...
+        while(rhs!=0)
+        {
+            pom = rhs;
+            rhs = lhs%rhs;
+            lhs = pom;	
+        }
+        
+        return lhs;
+    }
+}
```
Comment (nauka-programowania-MA) [inline homework/nwd-nnw/nwdNww.hpp:23]: Wrong indentation

File: homework/nwd-nnw/nwdNww.hpp [hunk 3]
```diff
@@ -1,11 +1,35 @@
...
+    int pom;
+    
+if(lhs == 0 && rhs == 0){
+    return 0;
+}    
+else{
+    lhs=abs(lhs);
+    rhs=abs(rhs);
+        
+        while(rhs!=0)
```
Comment (nauka-programowania-MA) [inline homework/nwd-nnw/nwdNww.hpp:14]: Add space between operations like '+' '-' '!=' '==' etc...
Add space before and after while, if, else etc...

File: homework/nwd-nnw/nwdNww.hpp [hunk 4]
```diff
@@ -1,11 +1,35 @@
...
+
+int NWW(int lhs, int rhs) {
+    // TODO: Implement me :)
+    if(lhs == 0 || rhs == 0){
+        return 0;
+    }
+    else{
+        lhs=abs(lhs);
+        rhs=abs(rhs);
+        return lhs*rhs/NWD(lhs, rhs);
```
Comment (nauka-programowania-MA) [inline homework/nwd-nnw/nwdNww.hpp:33]: Add space between operations like '+' '-' '!=' '==' etc...

**Review by nauka-programowania-MA — APPROVED** [review]

**Pull request closed without merging**